tests/acceptance/TestHelpers/AuthAppHelper.php

```diff
@@ -90,4 +90,12 @@ public static function deleteAppAuthToken(string $baseUrl, string $user, string
 			$password,
 		);
 	}
+
+	/**
+	 * @return string
+	 */
+	public static function getCreatedTokenRegex() : string {
```

Comment:
```suggestion
	public static function getCreatedAppTokenRegex() : string {
```

Reply:
What does this function do? I don't think this function returns an actual app token so we can name it in a way it says clearly what this function is for.